app/models/notification.rb

```diff
@@ -115,13 +115,13 @@ def remove_all_by_action_without_delay(notifiable_ids:, notifiable_type:, action
     def remove_all(notifiable_ids:, notifiable_type:)
       return unless %w[Article Comment Mention].include?(notifiable_type) && notifiable_ids.present?
 
-      Notifications::RemoveAllJob.perform_later(notifiable_ids, notifiable_type)
+      Notifications::RemoveAllWorker.perform_async(notifiable_ids, notifiable_type)
     end
 
     def remove_all_without_delay(notifiable_ids:, notifiable_type:)
       return unless %w[Article Comment Mention].include?(notifiable_type) && notifiable_ids.present?
 
-      Notifications::RemoveAllJob.perform_now(notifiable_ids, notifiable_type)
+      Notifications::RemoveAllWorker.new.perform(notifiable_ids, notifiable_type)
```

Comment:
I think this is the way to directly call a sidekiq job without putting it on a queue, but would be good to get additional eyes on it 😅